frontends/election-manager/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -43,116 +31,111 @@ const Header = styled.div`
-  ) => (event: React.ChangeEvent<HTMLSelectElement>) => void;
-  unadjudicateTranscription: (transcribedValue: string) => EventTargetFunction;
-}): JSX.Element {
-  const adjudicationQueue =
-    contestAdjudications.find((a) => a.value === '')?.transcriptions.length ||
-    0;
-
-  const adjudicationQueuePhrase = `${pluralize(
-    'transcriptions',
-    adjudicationQueue,
-    true
-  )} to adjudicate`;
-
-  const contestWriteInCount = contestAdjudications
-    .map((a) =>
-      a.transcriptions
-        .map((t) => t.count)
-        .flat()
-        .reduce(add, 0)
-    )
-    .reduce(add, 0);
+  updateAdjudication,
+}: Props): JSX.Element {
+  const contestWriteInCount = writeInSummaryEntries.reduce(
+    (acc, entry) => acc + entry.writeInCount,
+    0
+  );
 
-  const adjudicationValues: AdjudicationValue[] = [
-    ...contestAdjudications
-      .map((a) =>
-        a.transcriptions.map((t) => ({
-          name: t.value,
-          hasAdjudication: a.value !== t.value && a.value !== '',
-        }))
-      )
-      .flat(),
-    ...contest.candidates.map((c) => ({
-      name: `${c.name} (official candidate)`,
-      candidateId: c.id,
-      partyIds: c.partyIds,
-      hasAdjudication: false,
-    })),
-  ].sort((a, b) => a.name.localeCompare(b.name));
+  const adjudicatedGroups: AdjudicationGroup[] = [];
+  const pendingAdjudications: PendingWriteInAdjudication[] = [];
+
+  for (const [adjudicatedValue, entries] of groupBy(
+    writeInSummaryEntries,
+    (entry) => entry.writeInAdjudication?.adjudicatedValue
+  )) {
+    if (adjudicatedValue) {
+      adjudicatedGroups.push({
+        adjudicatedValue,
+        writeInCount: collections.reduce(
+          entries,
+          (acc, entry) => acc + entry.writeInCount,
+          0
+        ),
+        writeInAdjudications: Array.from(entries, (entry) => ({
+          id: entry.writeInAdjudication?.id as Id,
+          adjudicatedValue: entry.writeInAdjudication
+            ?.adjudicatedValue as string,
+          transcribedValue: entry.writeInAdjudication
+            ?.transcribedValue as string,
+          writeInCount: entry.writeInCount,
+        })),
+      });
+    } else {
+      for (const entry of entries) {
+        if (entry.transcribedValue) {
+          pendingAdjudications.push({
+            transcribedValue: entry.transcribedValue,
+            writeInCount: entry.writeInCount,
+          });
+        }
+      }
+    }
+  }
 
-  const hasAdjudicatedTranscriptions = contestAdjudications[0].value !== '';
+  const adjudicationQueuePhrase = `${format.count(
+    pendingAdjudications.length
+  )} ${pluralize('transcriptions', pendingAdjudications.length)} to adjudicate`;
+
+  const adjudicationValues = [
+    ...pendingAdjudications.map(
+      (pendingAdjudication): AdjudicationOption => ({
+        adjudicatedValue: pendingAdjudication.transcribedValue,
+        hasAdjudication: false,
+      })
+    ),
+    // TODO: we want to include some of these, I think
```

Comment:
wouldn't these all be covered by the transcribed values? At least for now until we let people enter custom values

Reply:
With this commented out it's only the _pending_ ones that will be shown. I'm honestly not sure what the intended behavior is.